### PR TITLE
feat: implement rate limiter in config

### LINKED ISSUE #182: setup-for-local-development throws 429 rate limiter error, should allow configuration or rate limit
Local node throws 429 rate limiter error:

```
Failed to load resource: the server responded with a status of 429 (Too Many Requests)
127.0.0.1:3000/org.dash.platform.dapi.v0.Platform/getDocuments:1 Failed to load resource: the server responded with a status of 429 (Too Many Requests)
app.js:8617 Something went wrong: NoAvailableAddressesForRetry: No available addresses for retry: Unknown Content-type received.
    at GrpcTransport.request (http://localhost:3331/_nuxt/vendors/app.js:5200:15)
    at async PlatformMethodsFacade.getDocuments (http://localhost:3331/_nuxt/vendors/app.js:4843:34)
```
Shumkov:
We removed rate limiter for local at some point but it seems it came back with new config templates refactoring. For now, you can just comment out rate limiter in nginx template.

We need to introduce  rateLimiter options to nginx section I believe and make it configurable.

https://discordapp.com/channels/670271785974890526/778903428876599316/780744832632225813

### PR DESCRIPTION
**Pull request by strophy**
This PR adds the nginx rate limiter to the config template.

## Issue being fixed or feature implemented
Resolves #182. Prior to use of templates, the local config did not enable the nginx rate limiter config. This was accidentally re-enabled in the templates, causing errors during testing. This PR allows the user to configure the rate limiter, and ensures it is off by default on in the local config.

## What was done?
Add enabled/rate/burst to config and nginx templates.

## How Has This Been Tested?
- [x] Tested against local network, rate limiter config is not included
- [x] Tested against evonet, rate limiter config is included

## Breaking Changes
None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/config/configJsonSchema.js [hunk 1]
```diff
@@ -172,6 +172,22 @@ module.exports = {
...
+                      minimum: 0,
+                    },
+                    burst: {
+                      type: 'integer',
+                      minimum: 0,
+                    },
+                  },
+                },
               },
               required: ['docker', 'http', 'grpc'],
```
Comment (shumkov) [inline src/config/configJsonSchema.js:192]: I would require `rateLimiter` for config consistency

File: src/config/configJsonSchema.js [hunk 2]
```diff
@@ -172,6 +172,22 @@ module.exports = {
...
+                    },
+                    rate: {
+                      type: 'integer',
+                      minimum: 0,
+                    },
+                    burst: {
+                      type: 'integer',
+                      minimum: 0,
+                    },
+                  },
```
Comment (shumkov) [inline src/config/configJsonSchema.js:189]: You forgot `required` and `additionalProperties: false`

File: src/config/configJsonSchema.js (hunk 1 above)
Comment (strophy) [reply src/config/configJsonSchema.js]: Good point, fixed!

File: src/config/configJsonSchema.js (hunk 2 above)
Comment (strophy) [reply src/config/configJsonSchema.js:189]: Good point, fixed!

**Review by shumkov — APPROVED** [review]
LGTM! 👍

**Pull request merged**